setup.py

```diff
@@ -24,25 +24,24 @@
       install_requires=[
           "requests>=2.31.0,<3",
           "pyjwt>=2,<3",
-          "arrow>=0.15.4,<2",
-          "gemd>=1.15.0,<2",
-          "boto3>=1.17.93,<2",
-          "botocore>=1.20.95,<2",
+          "arrow>=1.3.0,<2",
+          "gemd>=1.16.7,<2",
+          "boto3>=1.34.12,<2",
           "deprecation>=2.1.0,<3",
           "urllib3>=1.26.18,<3",
-          "tqdm>=4.62.3,<5"
+          "tqdm>=4.66.1,<5"
```

Comment:
Python 3.12 happily installs tqdm 1.0, and the feature we use (`tqdm.auto`) came in with 4.27.0.  I feel like we don't need to have a strong opinion on this one.
```suggestion
          "tqdm>=4.27.0,<5"
```

Reply:
Sure.